project.clj

```diff
@@ -126,13 +129,13 @@
              :dev-deps  {:dependencies [[org.bouncycastle/bcpkix-jdk18on]]}
              :dev [:defaults :dev-deps]
              :fips-deps {:dependencies [[org.bouncycastle/bcpkix-fips]
-                                       [org.bouncycastle/bc-fips]
-                                       [org.bouncycastle/bctls-fips]]
+                                        [org.bouncycastle/bc-fips]
+                                        [org.bouncycastle/bctls-fips]]
```

Comment:
These, and the ones below were out of alignment according to the editor, so I fixed it.